src/test/java/com/webank/webase/front/gm/pressure/PerformanceCollector.java

```diff
@@ -0,0 +1,111 @@
+package com.webank.webase.front.gm.pressure;
+
+import org.fisco.bcos.web3j.protocol.core.methods.response.TransactionReceipt;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class PerformanceCollector {
+  static Logger logger = LoggerFactory.getLogger(PerformanceCollector.class);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/WeBankFinTech/WeBASE-Front/pullRequest?prid=4823531)